root/dataframe/test_nested_rvec_snapshot.C

```diff
@@ -0,0 +1,55 @@
+#include <ROOT/RDataFrame.hxx>
+#include <ROOT/RVec.hxx>
+using namespace ROOT::VecOps;
+
+struct TwoInts {
+   int a, b;
+};
+
+struct TriggerVecOpsDictGeneration {
+   RVec<TwoInts> _1;
+   RVec<RVec<TwoInts>> _2;
+   RVec<RVec<int>> _3;
+   RVec<RVec<RVec<int>>> _4;
+};
```

Comment:
FYI, using _here_
```
#ifdef __ROOTCLING__
#prama link C++ class ROOT::VecOps::RVec<TwoInts>+;
...
```
would (should) have also worked.

Reply:
ah, did not realize that, that's better

Reply:
done